### DISCUSSION TITLE
如何自定义桌面快捷方式的名称？

### DISCUSSION THREAD

**Discussion by geeklibin** [tauri-apps/tauri#4655] [Q&A]
我希望我的桌面快捷方式的名称是中文的。
但是桌面快捷方式的名称似乎是使用的``tauri.conf.json``中`package.productName`，我无法用中文去赋值`package.productName`，因为使用中文会在构建时报错，同时我希望桌面的快捷方式是中文的。
请问我该如何解决这个问题。
非常感谢

**Comment by minosss** [comment]
在 mac 中 build 了下并不会报错。确保文件是 utf8?

**Reply by geeklibin** [reply]
tauri.conf.json：
```
package.productName = "中文名"

tauri.bundle.windows.wix.language = ["zh-CN"]
```
通过该方法解决了。

**Reply by 107295472** [reply]
uninstall 加到开始菜单的快捷方式怎么做，现在应用快捷方式默认有的，但卸载的没有

**Reply by kstar-er** [reply]
windows使用tauri.bundle.windows.wix.language = ["zh-CN"]这个配置没有问题，能显示中文名称。但是对于mac，中文名称build不会报错，再codesign的时候报错了，这个有没有什么解决方法呢
